### PR TITLE
Assign hazard class according to value mapping

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

**Comment by ismailsunni** [comment]
LGTM

File: safe/impact_function_v4/test/test_impact_function.py [hunk 1]
```diff
@@ -43,12 +43,15 @@ def read_json_flow(json_path):
...
 
-def run_scenario(scenario):
+def run_scenario(scenario, use_debug=False):
     """Run scenario
 
     :param scenario: Dictionary of hazard, exposure, and aggregation.
     :type scenario: dict
 
+    :param use_debug: If we should use debug when we run the scenario.
+    :rtype use_debug: bool
```
Comment (ismailsunni) [inline safe/impact_function_v4/test/test_impact_function.py]: type

File: safe/datastore/datastore.py [hunk 2]
```diff
@@ -43,6 +43,26 @@ def __init__(self, uri):
...
+        :rtype: bool
+        """
+        return self._use_index
+
+    @use_index.setter
+    def use_index(self, index):
+        """Setter if we use an index when we add a layer to the datastore.
+
+        :param index: A boolean if we use an index.
+        :rtype index: bool
```
Comment (ismailsunni) [inline safe/datastore/datastore.py]: type

File: safe/impact_function_v4/test/test_impact_function.py (hunk 1 above)
Comment (Gustry) [inline safe/impact_function_v4/test/test_impact_function.py]: fixed

File: safe/datastore/datastore.py (hunk 2 above)
Comment (Gustry) [inline safe/datastore/datastore.py]: fixed

**Comment by ismailsunni** [comment]
LGTM part 2 :D

**Pull request merged**